GameServer/gameobjects/GameNPC.cs

```diff
@@ -756,12 +760,58 @@ public virtual eFlags Flags
 			}
 		}
 
+		public bool IsGhost
+		{ get => (m_flags & eFlags.GHOST) != 0; }
+
+		public override bool IsStealthed
+		{ get => (m_flags & eFlags.STEALTH) != 0; }

```

Comment:
`m_flags.HasFlag( eFlags.STEALTH )` ?

Reply:
Hmm.  It might be better to use HasFlag than the accessors.

I'm happy either way.